app/models/generic_file.rb

```diff
@@ -7,6 +7,11 @@ class GenericFile < ActiveFedora::Base
   include Galtersufia::GenericFile::FullTextIndexing
   include Galtersufia::GenericFile::MimeTypes
 
+  PUBLIC_PERMISSION = "http://projecthydra.org/ns/auth/group#public"
```

Comment:
I think PUBLIC_PERMISSION should absolutely be a constant, but the other two might be better off ass functions? It's probably not a big deal since either way achieves roughly the same thing.